Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM goreleaser/goreleaser:latest as builder
+
+ENV GORELEASER_CURRENT_TAG=latest
+
+WORKDIR /tmp/cirrus-ci-agent
+ADD . /tmp/cirrus-ci-agent/
+
+RUN goreleaser --snapshot
+
+FROM gcr.io/distroless/base-debian10
+COPY --from=builder /tmp/cirrus-ci-agent/dist/agent_linux_amd64/agent /bin/cirrus-ci-agent
```

Comment:
Please add a [newline](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline) here, this way a simple `cat` and other Unix tools will be able to work with this file properly.